src/main/java/Commands/DukeCommand.java

```diff
@@ -0,0 +1,25 @@
+package Commands;
+
+import Exceptions.DukeException;
+import Tasks.Task;
+import Tasks.TaskList;
+import util.Storage;
+import util.Ui;
+
+import java.io.IOException;
+
+public class DukeCommand {
```

Comment:
I think a better practice should be making the `DukeCommand` an abstract class, that way I don't think there is a need to return 'Empty' for the default `execute()`, all the children of this class should implement the `execute()` method in there individually.